app/views/courses/preview/financial_support/_financial_support.html.erb

```diff
@@ -4,13 +4,11 @@
   <% if course.funding_type == 'salary' %>
     <%= render partial: 'courses/preview/financial_support/salaried' %>
   <% else %>
-    <p class="app-missing-section">
-      You can’t preview information about bursaries, scholarships and financial support, but it will be displayed in Find postgraduate teacher training.
-    </p>
+    <%= render partial: 'courses/preview/financial_support/scholarship_and_bursary' %>
```

Comment:
You probably better off porting across this 

https://github.com/DFE-Digital/search-and-compare-ui/blob/master/shared/Views/Shared/Components/CourseDetails/FinancialSupport/_FinancialSupport.cshtml